src/content/reference/platform-api/crd/configs.core.giantswarm.io.md

```diff
@@ -0,0 +1,418 @@
+---
```

Comment:
Never heard of this. We're using config maps for app configuration. Can this be removed?

Reply:
Done